themes/bootprint3/theme.config.php

```diff
@@ -1,4 +1,143 @@
 <?php
 return [
-    'extends' => 'bootstrap3'
+    'extends' => 'bootstrap3',
+    'icons' => [
+        'defaultSet' => 'Fugue',
```

Comment:
Do we really want to override the default set to Fugue here, or would it be safer to keep the default at FontAwesome, but override relevant icons with Fugue? I'm concerned that if we keep it like this, adding a new icon to bootstrap3 could cause a problem in bootprint3 since no corresponding Fugue entry will exist. Or in other words, a rule of thumb: child themes should keep the same default set as their parents to ensure that future inheritance does not have unexpected side effects.